fastlane/Fastfile.swift

```diff
@@ -12,35 +12,43 @@ class Fastfile: LaneFile {
 
     // MARK: - Code signing
 
-    func syncDevelopmentCodeSigningLane() {
+    func syncDevelopmentStagingCodeSigningLane() {
         desc("Sync the Development match signing for the Staging build")
         Match.syncCodeSigning(
             type: .development,
-            appIdentifier: [Constant.stagingBundleId]
+            environment: .staging
+        )
+    }
+    
+    func syncDevelopmentProductionCodeSigningLane() {
+        desc("Sync the Development match signing for the Staging build")
```

Comment:
here `production` environment is used, so please update the description:
desc("Sync the Development match signing for the Production build")

Reply:
Nice catch